modules/core/04-channel/types/events.go

```diff
@@ -8,11 +8,19 @@ import (
 
 // IBC channel events
 const (
-	AttributeKeyConnectionID       = "connection_id"
-	AttributeKeyPortID             = "port_id"
-	AttributeKeyChannelID          = "channel_id"
-	AttributeKeyChannelState       = "channel_state"
-	AttributeVersion               = "version"
+	AttributeKeyConnectionID   = "connection_id"
+	AttributeKeyPortID         = "port_id"
+	AttributeKeyChannelID      = "channel_id"
+	AttributeKeyChannelState   = "channel_state"
+	AttributeKeyVersion        = "version"
+	AttributeKeyConnectionHops = "connection_hops"
+	AttributeKeyOrdering       = "ordering"
+	AttributeKeyTimeout        = "timeout"
+
+	// upgrade specific keys
+	AttributeKeyUpgradeSequence = "sequence"
```

Comment:
I kept this as `...UpgradeSequence` because we already use `AttributeKeySequence` below for `packet_sequence`. I can renamed that one to `AttributeKeyPacketSequence` is people like that. Although then I should probably rename all the packet even attribute to start with `AttributeKeyPacket...`.